app/src/ui/changes/no-changes.tsx

```diff
@@ -697,8 +697,8 @@ export class NoChanges extends React.Component<
             <div className="text">
               <h1>No local changes</h1>
               <p>
-                You have no uncommitted changes in your repository! Here are
-                some friendly suggestions for what to do next.
+                There are no uncommitted changes to this repository. 
+                Here are some friendly suggestions on what to do next:
```

Comment:
Tentative feels on this change - `on` doesn't feel right, but `for` doesn't jump out to me as an excellent replacement:

```suggestion
                Here are some friendly suggestions for what to do next:
```

Reply:
does “as” sound better?

Reply:
> does “as to” sound better?

Not really. Ideally this would be as concise as possible, but the grammar nerd in me is unsatisfied with everything proposed so far...

Reply:
Maybe just “Here are some friendly suggestions:”?

Reply:
Yeah, I do not particularly like anything either. 

Perhaps "Your repository has no uncommitted changes." ...

Reply:
I changed it from “Your” as it might be someone else’s repo you have write access to. And we were focusing on the second bit about suggestions.

Reply:
Last suggestion for me: "This repository has no uncommitted changes. Here are some friendly suggestions:"

Reply:
Going to resolve this for now to step back and look at the overall message...